pkgs/development/python-modules/robotframework-debuglibrary/default.nix

```diff
@@ -0,0 +1,41 @@
+{ stdenv
+, buildPythonPackage
+, fetchPypi
+, prompt_toolkit
+, pygments
+, robotframework
+}:
+
+let
+  # prompt_toolkit version 1.x is required
+  prompt-toolkit-version-1 = prompt_toolkit.overrideAttrs (oldAttrs: rec {
+    version = "1.0.18";
+    src = fetchPypi {
+      inherit version;
+      pname = oldAttrs.pname;
+      sha256 = "dd4fca02c8069497ad931a2d09914c6b0d1b50151ce876bc15bde4c747090126";
+    };
+  });
```

Comment:
overriding versions in `python-modules` isn't allowed, as it can create incoherent package sets

Reply:
I see. I removed the pointed part.